src/site/content/en/blog/performance_tool_score_diffs.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Performance Tooling Score Differences
+subhead: I get different scores in Test My Site, PageSpee Insights, and Lighthouse - why?
+authors:
+  - egsweeny
+date: 2019-07-05
+hero: hero.jpg
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+alt: A description of the hero image for screen reader users.
+description: |
+  This post serves to explain the reasons that there are differences in performance scores between Test My Site (TMS), PageSpeed Insights (PSI), and Lighthouse. This document does not detail other differences in the tools, such as server run locations, network throttling presets, etc.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+# Why is my performance score different between different tools? What is my REAL score?
+
+<a href="https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics">Load is not a single moment in time</a>—it's an experience that no one metric can fully capture. There are multiple moments during the load experience that can affect whether a user perceives it as "fast", and if you just focus on one you might miss bad experiences that happen during the rest of the time. <b>This is why we generally use multiple metrics blended together to create our performance scores;</b> but different tools use different metrics and different weighted blends. This can be frustrating when you're looking for a source of truth about how fast your website is.
+
+This blog post serves to explain the reasons that there are differences in performance scores between three Google tools: <a href="https://www.thinkwithgoogle.com/intl/en-gb/feature/testmysite">Test My Site (TMS)</a>, <a href="https://developers.google.com/speed/pagespeed/insights/">PageSpeed Insights (PSI)</a>, and <a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse</a>. This document does <b>not</b> detail other differences in the tools, such as server run locations, network throttling presets, score variance between runs, etc.
+
+## Terminology
+
+Various terms about speed/performance are referenced in our tools. We want to start off with some clarification of how these terms are used.
+
+- <b>Page speed</b> refers to the speed of an individual page on your site (URL level).
+- <b>Site speed</b> refers to the speed of all of the pages on your site (origin level).
+
+It is important to note that ‘performance’ or ‘speed’ is not to be defined by a single metric (i.e. First Contentful Paint (FCP), Time to Interactive (TTI), etc.). This is motivated by several considerations, including:
+- (as mentioned before) Page "load is not a single moment in time—it's an experience that no one metric can fully capture" [<a href="https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics">source</a>]
```

Comment:
```suggestion
- (as mentioned before) Page "load" is not a single moment in time—it's an experience that no one metric can fully capture [source](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics)
```

I didn't realize this was a quote (I thought you had left off a closing quote). I wonder if it could be rephrased so the source is called out at the beginning. Maybe something like, "As mentioned in X, 'Load is not a...'"